### PR TITLE
Fix Android FTW and WAV

### PR DESCRIPTION
**Pull request by themuflon**

### CODE REVIEW

File: docs/examples/FileExample/src/FileExample.ls [hunk 1]
```diff
@@ -16,8 +16,8 @@ package
     public class FileExample extends Application
     {
 
-        // get a writable folder 
-        var writePath = Path.normalizePath(Path.getWritablePath() + "/Loom/Tests");
+        // get a writable folder
+        var writePath = Path.normalizePath(Path.getWritablePath() + "Loom/Tests");
```
Comment (bengarney) [inline docs/examples/FileExample/src/FileExample.ls]: Is this change needed for correctness?

Comment (themuflon) [inline docs/examples/FileExample/src/FileExample.ls]: Yeah, it made an extra / in the path, it worked everywhere, but still better to have it fixed.

**Comment by bengarney** [comment]
Have you run this on android devices and confirmed it works?

**Comment by themuflon** [comment]
Yes
